### DISCUSSION TITLE
Работает ли проект на Ubuntu?

### DISCUSSION THREAD

**Discussion by X1r0s** [Avdushin/GOAPPLE#3] [Q&A] [answered]
Привет друг, работает ли твой проект на Убунту?

**Comment by Avdushin** [comment, marked as answer]
Моё почтение.
Ранние версии проекта были написаны и для Ubuntu в том числе, но сейчас я решил временно убрать этот дистрибутив из списка.

**Reply by X1r0s** [reply]
thx
